### ISSUE TITLE
Remove Ammonia, Bleach, Lye etc from eat/drink menu

### ISSUE THREAD

**Issue by narkiel** [cataclysmbnteam/Cataclysm-BN#5701]
### Is your feature request related to a problem? Please describe.

There is no reason anyone would ever *want* to drink bleach etc.  At best, it's a roundabout redundant method for shift+Q.  Having them be drinkable just adds an unnecessary and unfun landmine to the eat/drink menu.

### Describe the solution you'd like

Remove the ability to eat/drink bleach, ammonia, lye, and any similar chemicals I may have be forgetting.

**Comment by AskaHope** [comment]
Add a flag like hazardous or something and remove those among other drugs from the "E" menu, they should be "a" only.

**Comment by blot-runner** [comment]
> Actually now that you mention it, there's already a flag to exclude dangerous substances from auto eat zones. We could potentially reuse that, although it would also exclude things like alcohol.

Why not just change the type from COMESTIBLES into AMMO (with ammo_type: "components)? This'll exclude it from the list of edible things completely.
For the record: this is exactly like salt water is coded in jsons.

**Comment by Proxiehunter** [comment]
> Actually now that you mention it, there's already a flag to exclude dangerous substances from auto eat zones. 

Not an expert but I'm pretty sure that doesn't prevent them from being (E)aten through the eat menu which is what @narkiel is talking about. And I'm pretty sure bleach, ammonia, etc. are _already_ excluded from auto eat zones or people would be accidentally drinking bleach more often.

**Comment by blot-runner** [comment]
> Can you use non-comestibles as a component _of_ a comestible?

Yes. For example, salt water (non-comestible) is used as a component for crafting bleach (at present a comestible).

**Comment by Ignaramico** [comment]
doesnt eating that kind of stuff prompt a warning?, 
and iirc in the earliers days of CDDA forum was a list of challenges and one of them was surviving eating a gallon of bleach (among other ones) lol :x,

**Comment by blot-runner** [comment]
> doesnt eating that kind of stuff prompt a warning?,

As a matter of fact, it does, for both bleach and ammonia. Ditto for liquid fertilizer and sewage sample, but not for vodka, which also has UNSAFE_CONSUME flag. IDK, maybe that's enough of a safeguard?
By the way, lye is listed as a possible food, but I was unable to drink it, choosing the entry simply does nothing. Talk about disturbing topic for a bug report))

**Comment by RoyalFox2140** [comment]
As another source of YASD we don't want drinkable chemicals so it's considered a bug.

**Comment by RoyalFox2140** [comment]
> Can you use non-comestibles as a component _of_ a comestible?

You can make oranges from dirt and transmutation runes in 2200.

**Comment by Ignaramico** [comment]
> By the way, lye is listed as a possible food, but I was unable to drink it, choosing the entry simply does nothing. Talk about disturbing topic for a bug report))

wasnt that the same issue with paper and other stuff(like seeds irrc)? being listed as food on the eat menu but not able to eat it
